mesh/mesh_test.go

```diff
@@ -404,6 +404,20 @@ func TestProcessLayer(t *testing.T) {
 				},
 			},
 		},
+		{
+			"missing but can make progress",
+			[]call{
+				{
+					updates: rlayers(
+						rlayer(start, rblock(idg("1"), fixture.Valid(), fixture.Data())),
+						rlayer(start+1, rblock(idg("2"), fixture.Valid(), fixture.Data())),
+						rlayer(start+2, rblock(idg("3"), fixture.Valid())),
```

Comment:
maybe extend it with one more block with data? and one more call that both got applied

Reply:
done